[debug] [2019-04-09T20:14:30.996Z] ----------------------------------------------------------------------
[debug] [2019-04-09T20:14:30.998Z] Command:       /usr/local/bin/node /usr/local/bin/firebase serve
[debug] [2019-04-09T20:14:30.998Z] CLI Version:   6.5.0
[debug] [2019-04-09T20:14:30.998Z] Platform:      darwin
[debug] [2019-04-09T20:14:30.999Z] Node Version:  v11.6.0
[debug] [2019-04-09T20:14:31.005Z] Time:          Tue Apr 09 2019 23:14:30 GMT+0300 (Eastern European Summer Time)
[debug] [2019-04-09T20:14:31.006Z] ----------------------------------------------------------------------
[debug] 
[debug] [2019-04-09T20:14:31.021Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2019-04-09T20:14:31.022Z] > authorizing via signed-in user
[debug] [2019-04-09T20:14:31.022Z] [iam] checking project whac-a-mole-677b1 for permissions ["firebase.projects.get"]
[debug] [2019-04-09T20:14:31.023Z] > refreshing access token with scopes: ["email","https://www.googleapis.com/auth/cloud-platform","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","openid"]
[debug] [2019-04-09T20:14:31.023Z] >>> HTTP REQUEST POST https://www.googleapis.com/oauth2/v3/token  
 <request body omitted>
[debug] [2019-04-09T20:14:31.319Z] <<< HTTP RESPONSE 200
[debug] [2019-04-09T20:14:31.341Z] >>> HTTP REQUEST POST https://cloudresourcemanager.googleapis.com/v1/projects/whac-a-mole-677b1:testIamPermissions  

[debug] [2019-04-09T20:14:32.407Z] <<< HTTP RESPONSE 200
[debug] [2019-04-09T20:14:32.408Z] >>> HTTP REQUEST GET https://cloudresourcemanager.googleapis.com/v1/projects/whac-a-mole-677b1  
 
[debug] [2019-04-09T20:14:33.327Z] <<< HTTP RESPONSE 200
[info] 
[info] === Serving from '/Users/tristankrass/Desktop/geeksOnWheelsLab'...
[info] 
[debug] [2019-04-09T20:14:34.611Z] Error: EACCES: permission denied, mkdir '/usr/local/lib/node_modules/firebase-tools/node_modules/@google-cloud/functions-emulator/logs'
    at Object.mkdirSync (fs.js:775:3)
    at Object.assertLogsPath (/usr/local/lib/node_modules/firebase-tools/node_modules/@google-cloud/functions-emulator/src/emulator/logs.js:65:10)
    at new Controller (/usr/local/lib/node_modules/firebase-tools/node_modules/@google-cloud/functions-emulator/src/cli/controller.js:77:32)
    at FunctionsEmulator.start (/usr/local/lib/node_modules/firebase-tools/lib/functionsEmulator.js:102:23)
    at Object.start (/usr/local/lib/node_modules/firebase-tools/lib/serve/functions.js:7:33)
    at /usr/local/lib/node_modules/firebase-tools/lib/serve/index.js:15:23
    at arrayMap (/usr/local/lib/node_modules/firebase-tools/node_modules/lodash/lodash.js:639:23)
    at Function.map (/usr/local/lib/node_modules/firebase-tools/node_modules/lodash/lodash.js:9556:14)
    at _serve (/usr/local/lib/node_modules/firebase-tools/lib/serve/index.js:13:26)
    at Command.module.exports.Command.description.option.option.option.option.before.action [as _action] (/usr/local/lib/node_modules/firebase-tools/lib/commands/serve.js:55:12)
[error] 
[error] Error: An unexpected error has occurred.
